src/tests/ftest/util/ior_test_base.py

```diff
@@ -413,11 +413,8 @@ def _execute_command(self, command, fail_on_err=True, display_output=True, hosts
         if hosts is None:
             hosts = self.hostlist_clients
         result = pcmd(hosts, command, verbose=display_output, timeout=300)
-        if 0 not in result and fail_on_err:
-            hosts = [str(
-                nodes) for code, nodes in list(
-                    result.items()) if code != 0]
-            raise CommandFailure(
-                "Error running '{}' on the following hosts: {}".format(
-                    command, NodeSet(",".join(hosts))))
+        if (0 not in result or len(result) > 1) and fail_on_err:
+            hosts = [str(nodes) for code, nodes in list(result.items()) if code != 0]
+            raise CommandFailure("Error running '{}' on the following hosts: {}".format(
+                command, NodeSet(",".join(hosts))))
```

Comment:
This shouldn't break anything, but as a sanity check:
`Test-tag: ior`

Reply:
In addition to the above I assume?